config/ur16e/visual_parameters.yaml

```diff
@@ -7,92 +7,117 @@ mesh_files:
       mesh:
         package: ur_description
         path: meshes/ur10e/visual/base.dae
-      material:
-        name: "LightGrey"
-        color: "0.7 0.7 0.7 1.0"
     collision:
       mesh:
         package: ur_description
         path: meshes/ur10e/collision/base.stl
+    mesh_offset:
+      x: 0.0
+      y: 0.0
+      z: 0.0
+      roll: 0.0
+      pitch: 0.0
+      yaw: !degrees 180
 
   shoulder:
     visual:
       mesh:
         package: ur_description
         path: meshes/ur10e/visual/shoulder.dae
-      material:
-        name: "LightGrey"
-        color: "0.7 0.7 0.7 1.0"
     collision:
       mesh:
         package: ur_description
         path: meshes/ur10e/collision/shoulder.stl
+    mesh_offset:
+      x: 0.0
+      y: 0.0
+      z: 0.0
+      roll: 0.0
+      pitch: 0.0
+      yaw: !degrees 180
 
   upper_arm:
     visual:
       mesh:
         package: ur_description
         path: meshes/ur16e/visual/upperarm.dae
-      material:
-        name: "LightGrey"
-        color: "0.7 0.7 0.7 1.0"
     collision:
       mesh:
         package: ur_description
         path: meshes/ur16e/collision/upperarm.stl
       mesh_files:
+    mesh_offset:
+      x: 0.0
+      y: 0.0
+      z: 0.1762
+      roll: !degrees 90
+      pitch: 0.0
+      yaw: !degrees -90
 
   forearm:
     visual:
       mesh:
         package: ur_description
         path: meshes/ur16e/visual/forearm.dae
-      material:
-        name: "LightGrey"
-        color: "0.7 0.7 0.7 1.0"
     collision:
       mesh:
         package: ur_description
         path: meshes/ur16e/collision/forearm.stl
+    mesh_offset:
+      x: 0.0
+      y: 0.0
+      z: 0.0393
```

Comment:
The elbow offset was 0.04 in the physical parameters file and not 0.0393 is it intended to change the value?